app/views/shared/_navbar.html.erb

```diff
@@ -1,59 +1,63 @@
 <div class="navbar-wagon navbar-wagon-fixed">
-  <!-- Logo -->
-  <a href="/" class="navbar-wagon-brand">
-    <%= image_tag "unpolarise.svg" %>
-  </a>
+
+  <!-- Left Navigation -->
+  <div class="navbar-wagon-left hidden-xs hidden-sm">
+    <!-- Logo -->
+    <%= link_to root_path, class: "navbar-wagon-brand" do %>
+      <%= image_tag "unpolarise.svg" %>
+    <% end %>
+    <!-- Search form: The search form appears only if the user is logged in -->
+    <% if user_signed_in? %>
+      <form action="" class="navbar-wagon-search">
+        <input type="text" class="navbar-wagon-search-input" placeholder="What are you interested in?">
+        <button type="submit" class="navbar-wagon-search-btn">
+          <i class="fa fa-search"></i>
+        </button>
+      </form>
+    <% end %>
+  </div>
 
   <!-- Right Navigation -->
   <div class="navbar-wagon-right hidden-xs hidden-sm">
+    <% if user_signed_in? %>
+      <!-- Link to Articles owned by the user -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Articles</a>
 
-    <!-- Search form -->
-    <!--
-    <form action="" class="navbar-wagon-search">
-      <input type="text" class="navbar-wagon-search-input" placeholder="Search something">
-      <button type="submit" class="navbar-wagon-search-btn">
-        <i class="fa fa-search"></i>
-      </button>
-    </form> -->
-
-    <!-- Text link -->
-    <a href="" class="navbar-wagon-item navbar-wagon-link">Share your Articles</a>
-
-    <!-- Text link -->
-    <a href="" class="navbar-wagon-item navbar-wagon-link">Help</a>
+      <!-- Link to Articles the user has booked previously -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Bookings</a>
 
-    <!-- Login text link -->
-    <%= link_to "Sign Up", new_user_registration_path, class: "navbar-wagon-item navbar-wagon-link" %>
+      <!-- Link to our Help page -->
+      <a href="" class="navbar-wagon-item navbar-wagon-link">Help</a>
 
-    <!-- Login text link -->
-    <%= link_to "Login", new_user_session_path, class: "navbar-wagon-item navbar-wagon-link" %>
-
-    <!-- Notification link-->
-    <!--
-    <a href="" class="navbar-wagon-item navbar-wagon-link">
-      <div class="badge-container">
-        <i class="fa fa-envelope-o"></i>
-        <div class="badge badge-blue">3</div>
+      <!-- Profile picture and dropdown -->
+      <div class="navbar-wagon-item">
+        <div class="dropdown">
+          <img src="http://placehold.it/50x50" class="avatar dropdown-toggle" id="navbar-wagon-menu" data-toggle="dropdown">
+          <ul class="dropdown-menu dropdown-menu-right navbar-wagon-dropdown-menu">
+            <li><a href="#">Profile</a></li>
+            <li><a href="#">Dashboard</a></li>
+            <li>
+              <%= link_to destroy_user_session_path, method: :delete do %>
```

Comment:
Suggestion to simplify the 3 lines 40-42:
<%= link_to "Log out", destroy_user_session_path, method: :delete %>